tools/file_packager.py

```diff
@@ -655,35 +669,10 @@ def generate_js(data_target, data_files, metadata):
         new DataRequest(files[i]['start'], files[i]['end'], files[i]['audio'] || 0).open('GET', files[i]['filename']);
       }\n''' % (create_preloaded if options.use_preload_plugins else create_data)
 
-  if options.has_embedded:
-    if options.obj_output:
-      if options.wasm64:
-        code += '''\
```

Comment:
so does this code go directly into the wrapper JS code? and this PR is moving it to regular library code?

Reply:
Yes, it moved from pre-js code that would run before `__wasm_call_ctors` into a JS library function that gets called from a ctor at priority 49.